<launch>
    <group ns="robot_other1">
	    <node pkg="reinovo_control" type="blank" name="blank" output="screen" required="true"/>
	</group>
	<!--以上勿动-->

	<include file="$(find arm_controller)/launch/pick_ar.launch" /> 
	<include file="$(find ar_pose)/launch/ar_pick.launch" /> 


</launch>
